include/wasmtime.hh

```diff
@@ -1556,8 +1556,9 @@ public:
    * This function is the same as wasmtime_module_deserialize except that it
```

Comment:
Oh I forgot about this earlier but could this be updated to link to the `deserialize` method?

Reply:
Same same, will do.